src/CoreFx.Private.TestUtilities/src/System/Diagnostics/RemoteExecutorTestBase.Process.cs

```diff
@@ -49,13 +50,28 @@ private static RemoteInvokeHandle RemoteInvoke(MethodInfo method, string[] args,
             }
 
             // If we need the host (if it exists), use it, otherwise target the console app directly.
-            string testConsoleAppArgs = "\"" + a.FullName + "\" " + t.FullName + " " + method.Name + " " + string.Join(" ", args);
+            var testConsoleAppArgs = new StringBuilder();
+            testConsoleAppArgs.Append(ExtraParameter);
```

Comment:
This might be cleaner if you created an "AppendQuoted" method .. it could also escape existing quotes.